service/src/main/java/com/ge/predix/acs/attribute/readers/AttributesWrapper.java

```diff
@@ -0,0 +1,35 @@
+package com.ge.predix.acs.attribute.readers;
+
+import java.util.Set;
+
+import com.ge.predix.acs.model.Attribute;
+
+public class AttributesWrapper {
+
+    public AttributesWrapper(final Set<Attribute> attributes) {
+        this.attributes = attributes;
+    }
+
+    public AttributesWrapper(final RuntimeException exception) {
+        this.exception = exception;
+    }
+
+    private RuntimeException exception;
```

Comment:
* Consider using a `state` field which would return a set of well known states, instead of exception.
* minor -  pls move the field declarations to the top